web/src/features/fba/fireCenterExtents.ts

```diff
@@ -0,0 +1,50 @@
+export const extentsMap = new Map([
```

Comment:
I'd like to see a bit more documentation here. e.g., I'm guessing the coordinates are in BC Albers? What did you do to calculate the extents?